### ISSUE TITLE
[A11y Switch]: Tooltip doesn't show on keyboard focus

### ISSUE THREAD

**Issue by mari-shakhanava** [epam/UUI#2959]
## Description

Tooltip component doesn't display additional content when Switch/Checkbox 
receives keyboard focus, only on mouse hover. This violates [WCAG 1.4.13 Content on Hover or Focus (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html).

## Steps to Reproduce

https://codesandbox.io/p/sandbox/uui-switch-isssue-exapmle-9dwn8y

## Actual result

- Hover:  Tooltip shows
- Keyboard Focus (Tab):  Tooltip doesn't show
Focus events on nested `<input>` inside Switch/Checkbox don't bubble to the wrapper element that Tooltip monitors.

## Expected result

Both hover AND focus should trigger the tooltip.

## Environment
* UUI version: 6.3.3

**Comment by JuliaMV** [comment]
Released in v6.4.4.
